pulp_rpm/app/models/advisory.py

```diff
@@ -222,7 +237,7 @@ class UpdateCollection(BaseModel):
 
     name = models.TextField()
     shortname = models.TextField()
-    module = models.TextField(default='')
+    module = JSONField(null=True)
```

Comment:
Actually -- this has a defined schema (NSVCA) and it's only ever one of them.  Really, we should probably just use a foreign key to a new model.

Agree/disagree?  Anyways, it still makes sense to not store JSON in TextField.